### PR TITLE
feat: allow passing a certification payload

### PR DESCRIPTION
**Pull request by betodealmeida**
Allow passing a certification payload when syncing datasets from dbt.

### CODE REVIEW

File: src/preset_cli/cli/superset/sync/dbt/datasets.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ def sync_datasets(  # pylint: disable=too-many-locals, too-many-branches, too-ma
     database: Any,
     disallow_edits: bool,
     external_url_prefix: str,
+    certification=None,
```
Comment (zephyring) [inline src/preset_cli/cli/superset/sync/dbt/datasets.py:76]: can we specify the type hint here

File: tests/cli/superset/sync/dbt/datasets_test.py [hunk 2]
```diff
@@ -275,6 +275,65 @@ def test_sync_datasets_no_metrics(mocker: MockerFixture) -> None:
...
+    }
+
+    sync_datasets(
+        client=client,
+        models=models,
+        metrics=[],
+        database={"id": 1, "sqlalchemy_uri": "postgresql://user@host/examples_dev"},
+        disallow_edits=False,
+        external_url_prefix="",
+        certification={"details": "This dataset is synced from dbt Cloud"},
```
Comment (zephyring) [inline tests/cli/superset/sync/dbt/datasets_test.py:296]: do we already have another test case when cert is None

Comment (betodealmeida) [reply tests/cli/superset/sync/dbt/datasets_test.py:296]: We do!

File: src/preset_cli/cli/superset/sync/dbt/datasets.py (hunk 1 above)
Comment (betodealmeida) [reply src/preset_cli/cli/superset/sync/dbt/datasets.py:76]: Ah, I forgot! Sorry, will fix!

**Review by zephyring — APPROVED** [review]
